sdk/keyvault/keyvault-keys/recordings/browsers/cryptographyclient_all_decrypts_happen_remotely/recording_before_all_hook.json

```diff
@@ -904,46 +904,599 @@
     "x-content-type-options": "nosniff",
-    "date": "Fri, 04 Oct 2019 23:30:48 GMT",
+    "date": "Tue, 15 Oct 2019 21:32:50 GMT",
     "p3p": "CP=\"DSP CUR OTPi IND OTRi ONL FIN\"",
-    "x-ms-request-id": "2abc0da7-adbc-4832-8b87-693e11051e00",
+    "x-ms-request-id": "1df65d4f-0f50-4d3b-9132-dc7da4d50000",
     "cache-control": "no-cache, no-store",
     "content-type": "application/json; charset=utf-8",
     "content-length": "1231",
-    "x-ms-ests-server": "2.1.9485.10 - SCUS ProdSlices",
+    "x-ms-ests-server": "2.1.9524.11 - SCUS ProdSlices",
     "referrer-policy": "strict-origin-when-cross-origin",
     "expires": "-1"
    }
   },
   {
-   "method": "DELETE",
+   "method": "GET",
+   "url": "https://keyvault_name.vault.azure.net/deletedkeys/cryptography-client-test",
+   "query": {
+    "api-version": "7.0"
+   },
+   "requestBody": null,
+   "status": 404,
+   "response": "{\"error\":{\"code\":\"KeyNotFound\",\"message\":\"Deleted Key not found: cryptography-client-test\"}}",
+   "responseHeaders": {
+    "strict-transport-security": "max-age=31536000;includeSubDomains",
+    "x-content-type-options": "nosniff",
+    "x-ms-keyvault-network-info": "addr=52.170.185.229;act_addr_fam=InterNetwork;",
+    "x-aspnet-version": "4.0.30319",
+    "x-powered-by": "ASP.NET",
+    "status": "404",
+    "x-ms-keyvault-region": "westus",
+    "content-length": "108",
+    "pragma": "no-cache",
+    "server": "Microsoft-IIS/10.0",
+    "x-ms-keyvault-service-version": "1.1.0.879",
+    "date": "Tue, 15 Oct 2019 21:32:50 GMT",
+    "content-type": "application/json; charset=utf-8",
+    "x-ms-request-id": "3425d833-04b5-4688-be5c-ab9c7d7e0fba",
+    "cache-control": "no-cache",
+    "expires": "-1"
+   }
+  },
+  {
+   "method": "GET",
+   "url": "https://keyvault_name.vault.azure.net/deletedkeys/cryptography-client-test",
+   "query": {
+    "api-version": "7.0"
+   },
+   "requestBody": "",
+   "status": 401,
+   "response": "{\"error\":{\"code\":\"Unauthorized\",\"message\":\"Request is missing a Bearer or PoP token.\"}}",
+   "responseHeaders": {
+    "strict-transport-security": "max-age=31536000;includeSubDomains",
+    "www-authenticate": "Bearer authorization=\"https://login.windows.net/72f988bf-86f1-41af-91ab-2d7cd011db47\", resource=\"https://vault.azure.net\"",
+    "x-ms-keyvault-network-info": "addr=52.170.185.229;act_addr_fam=InterNetwork;",
+    "x-aspnet-version": "4.0.30319",
+    "x-powered-by": "ASP.NET",
+    "status": "401",
+    "x-ms-keyvault-region": "westus",
+    "content-length": "87",
+    "pragma": "no-cache",
+    "server": "Microsoft-IIS/10.0",
+    "x-ms-keyvault-service-version": "1.1.0.879",
+    "date": "Tue, 15 Oct 2019 21:32:51 GMT",
+    "content-type": "application/json; charset=utf-8",
+    "x-ms-request-id": "c4e3956f-0fa0-477c-9577-beebe983e6d0",
+    "cache-control": "no-cache",
+    "x-content-type-options": "nosniff",
+    "expires": "-1"
+   }
+  },
+  {
+   "method": "POST",
```

Comment:
Odd to find this here in the middle. Did the token expire? Is your recorded test infrastructure effectively ignoring all this at this point anyway, or could this cause a problem?

Reply:
This won't cause a problem!